scss/typography/_helpers.scss

```diff
@@ -75,4 +108,15 @@ $stat-font-size: 2.5rem !default;
     margin-#{$global-left}: 0;
     list-style: none;
   }
+  
+  // Use to create an initial aka dropcap
+  .dropcaps{
+    line-height: 1.5; // Disabling default line height of an element
+    &:first-letter {
+      @include foundation-dropcaps;
+    }
```

Comment:
The "dropcaps" component is the full paragraph. `@mixin foundation-dropcaps` should contain `&:first-letter {...}`.